src/components/_checkbox.scss

```diff
@@ -0,0 +1,123 @@
+@use "../design-tokens/color";
+@use "../design-tokens/typography";
+
+$check-box-size: 1rem;
+$check-background: '::before';
+$check-icon: '::after';
+$check-ease-out-exponential: cubic-bezier(0.04, 0.08, 0.5, 0.95);
+
+/* stylelint-disable selector-max-specificity */
+
+@mixin Checkbox {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  cursor: pointer;
+
+  // ----------
+  //  Elements
+  // ----------
+
+  label {
+    @include typography.Typography__SmallScreenBody;
```

Comment:
Oh this needs to be updated off `master` (`git fetch && git rebase origin/master`). It’s `Typography--*` now.